common/test/java/com/couchbase/lite/BaseCollectionTest.kt

```diff
@@ -64,5 +66,15 @@ open class BaseCollectionTest : BaseDbTest() {
         Assert.assertEquals(doc.id, savedDoc!!.id)
         return savedDoc
     }
+
+    @Throws(CouchbaseLiteException::class)
+    protected fun createDocsInBaseCollectionTest(n: Int) {
+        for (i in 0 until n) {
+            val doc = MutableDocument(String.format(Locale.US, "doc_%03d", i))
+            doc.setValue("key", i)
+            saveDocInBaseCollectionTest(doc)
```

Comment:
Is the "Base" collection the default collection?

Reply:
No, it's the test collection that is created in BaseCollectionTest, but I'll change the name of this function to avoid confusion

Reply:
I see.  Cool!